tests/rados/test_osd_scrub_load_threshold_parameter.py

```diff
@@ -0,0 +1,548 @@
+"""
+Polarion ID: CEPH-83620201 - Verification of the osd_scrub_load_threshold parameter functionality
+
+The file contain the method to verify the osd_scrub_load_threshold parameter.
+The following steps are verified-
+1. Verification of the default value of the osd_scrub_load_threshold parameter, which is  10.0.
+
+2. Modification of the osd_scrub_load_threshold value to 0 and verification that the scrub operation does not proceed.
+   The expected log message is:
+     osd-scrub:scrub_load_below_threshold: loadavg .* >= max .* = no.
+
+3. Creation of an inconsistent object and confirmation that the repair operation does not occur.
+
+4. Modification of the osd_scrub_load_threshold value from 0 back to its default value.
+   Verification that the scrub operation resumes and the inconsistent object is repaired.
+   The expected log message is:
+     osd-scrub:scrub_load_below_threshold: loadavg .* >= max .* = yes.
```

Comment:
please add a method to calculate the load on the cluster, and then we can  definitively say if the scrub operations should resume or not!

please comment the scenario 4 from the test until then.

Reply:
The scenario is commented.